### PR TITLE
Update cartodb-postgres extension to 0.35.0

### PR DESCRIPTION
**Pull request by esloho**
Related to https://github.com/CartoDB/support/issues/2273

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ Development
 ### Bug fixes / enhancements
 - None yet
 
+4.32.1 (2019-12-30)
```
Comment (gonzaloriestra) [inline NEWS.md]: Do you need to release a new version for any reason? Otherwise, you should place the notice above, in the Development section.

Comment (esloho) [reply NEWS.md]: If I'm not wrong I need to release a new version in order to get new users to have the new extension version. Isn't it?

Comment (gonzaloriestra) [reply NEWS.md]: There's no need for that, new users will have it with the changes on DBService. This cartodb version is mainly used for on-premises.

File: NEWS.md [hunk 2]
```diff
@@ -2,7 +2,11 @@ Development
 -----------
 
 ### NOTICES
-- None yet
+* This release upgrades the CartoDB PostgreSQL extension to `0.35.0`. Run the following to have it available:
```
Comment (gonzaloriestra) [inline NEWS.md]: We must use hyphens in this section :)

```suggestion
- This release upgrades the CartoDB PostgreSQL extension to `0.35.0`. Run the following to have it available:
```

**Comment by esloho** [comment]
Retest this please

**Pull request merged**